YCloudYUSA/y_lb_demo_content
# Init demo content

Jira issues: 
- [DS-1042](https://yusa.atlassian.net/browse/DS-1042)
- [DS-1097](https://yusa.atlassian.net/browse/DS-1097)
-  [DS-1043](https://yusa.atlassian.net/browse/DS-1043)
-  [DS-1044](https://yusa.atlassian.net/browse/DS-1044)
-  [DS-1045](https://yusa.atlassian.net/browse/DS-1045)
-  [DS-1046](https://yusa.atlassian.net/browse/DS-1046)
-  [DS-1047](https://yusa.atlassian.net/browse/DS-1047)
-  [DS-1048](https://yusa.atlassian.net/browse/DS-1048)
-  [DS-1088](https://yusa.atlassian.net/browse/DS-1088)
- [DS-1089](https://yusa.atlassian.net/browse/DS-1089)
- [DS-1090](https://yusa.atlassian.net/browse/DS-1090)
- [DS-1091](https://yusa.atlassian.net/browse/DS-1091)
- [DS-1092](https://yusa.atlassian.net/browse/DS-1092)
- [DS-1093](https://yusa.atlassian.net/browse/DS-1093)
- [DS-1094](https://yusa.atlassian.net/browse/DS-1094)
- [DS-1095](https://yusa.atlassian.net/browse/DS-1095)
- [DS-1096](https://yusa.atlassian.net/browse/DS-1096)
- [DS-1098](https://yusa.atlassian.net/browse/DS-1098)


How to check:

```sh
drush en y_lb_init_demo_content -y
drush migrate:import --group y_lb_init_demo_content
```


 - go to admin/modules
 - enable the Y Layout Builder - Init Demo Pages module
 - go to /admin/structure/migrate/manage/y_lb_init_demo_content/migrations
 - run this migration /admin/structure/migrate/manage/y_lb_init_demo_content/migrations/y_lb_init_demo_media_image/execute
 - run these migrations : 
    -  /admin/structure/migrate/manage/y_lb_init_demo_content/migrations/y_lb_init_demo_homepage/execute
    - /admin/structure/migrate/manage/y_lb_init_demo_content/migrations/y_lb_init_demo_page_who_we_are/execute
    - /admin/structure/migrate/manage/y_lb_init_demo_content/migrations/y_lb_init_demo_page_programs_overview/execute
    - /admin/structure/migrate/manage/y_lb_init_demo_content/migrations/y_lb_init_demo_page_programs_category/execute
 -  as alternative you can run this drush command: `drush migrate:import --group y_lb_init_demo_content`
 - go to /admin/content
 - visit all page with prefix in the title " Init Demo"
 - verify that pages looks like on design
